pages/data/products/normal-ozone-maps.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <v-container>
+    <h1>{{ $t('data.products.normal-ozone-maps.title') }}</h1>
+    <v-btn-toggle v-model="ozoneMapTab" mandatory>
+      <v-btn value="global">
+        {{ $t('data.products.normal-ozone-maps.global') }}
+      </v-btn>
+      <v-btn value="northern">
+        {{ $t('data.products.normal-ozone-maps.northern') }}
+      </v-btn>
+      <v-btn value="southern">
+        {{ $t('data.products.normal-ozone-maps.southern') }}
+      </v-btn>
+    </v-btn-toggle>
+
+    <section v-if="ozoneMapTab === 'global'">
+      <h2>{{ $t('data.products.normal-ozone-maps.globalDesc') }}</h2>
+      <graph-carousel
+        :key="Object.keys(normalOzoneMapsGlobal)"
+        :graphs="normalOzoneMapsGlobal"
+      ></graph-carousel>
+    </section>
+
+    <section v-if="ozoneMapTab === 'northern'">
+      <h2>{{ $t('data.products.normal-ozone-maps.northernDesc') }}</h2>
+      <graph-carousel
+        :key="Object.keys(normalOzoneMapsNorthern)"
+        :graphs="normalOzoneMapsNorthern"
+      ></graph-carousel>
+    </section>
+
+    <section v-if="ozoneMapTab === 'southern'">
+      <h2>{{ $t('data.products.normal-ozone-maps.southernDesc') }}</h2>
+      <graph-carousel
+        :key="Object.keys(normalOzoneMapsSouthern)"
+        :graphs="normalOzoneMapsSouthern"
+      ></graph-carousel>
+    </section>
+  </v-container>
+</template>
+
+<script>
+import GraphCarousel from '~/components/GraphCarousel'
+export default {
+  components: {
+    'graph-carousel': GraphCarousel,
+  },
+  data() {
+    return {
+      baseOzoneClimatologyURL: `${this.$config.WOUDC_UI_WAF_URL}/products/ozone_maps/climatology/`,
+      ozoneMapTab: 'global',
+      normalozoneMapTypes: ['global', 'northern', 'southern'],
+    }
+  },
+  head() {
+    return {
+      title: this.$t('data.products.normal-ozone-maps.title'),
+      titleTemplate: this.$titleTemplate(
+        this.$t('common.woudc'),
+        this.$t('common.woudcFull')
+      ),
+      meta: [
+        {
+          hid: 'description',
+          name: 'description',
+          content: this.$t('data.products.normal-ozone-maps.description'),
+        },
+        {
+          hid: 'keywords',
+          name: 'keywords',
+          content: this.$t('data.products.normal-ozone-maps.keywords'),
+        },
+      ],
+    }
+  },
+  computed: {
+    normalOzoneMapsGlobal() {
+      const { baseOzoneClimatologyURL } = this
+      const months = [
+        'jan',
+        'feb',
+        'mar',
+        'apr',
+        'may',
+        'jun',
+        'jul',
+        'aug',
+        'sept',
+        'oct',
+        'nov',
+        'dec',
+      ]
```

Comment:
To avoid repetition, I think this can be placed in data() instead. Then you can simply reference this.months when you need to use it.